tutorials/Basics/Methods-and-Parameters-for-Linking.ipynb

```diff
@@ -0,0 +1,1455 @@
+       "<Figure size 1152x432 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "track = tobac.themes.tobac_v1.linking_trackpy(\n",
+    "    features, data, dt=dt, dxy=dxy, v_max=100, cell_number_start=0\n",
+    ")\n",
+    "\n",
+    "fig, ax = plt.subplots(ncols=1, nrows=1, figsize=(16, 6))\n",
+    "\n",
+    "data.isel(time=50).plot(ax=ax, cmap=\"Greys\")\n",
+    "\n",
+    "for i, cell_track in track.groupby(\"cell\"):\n",
+    "\n",
+    "    index = int(cell_track[\"cell\"].data[0])\n",
+    "\n",
+    "    cell_track.plot.scatter(\n",
+    "        y=\"projection_x_coordinate\",\n",
+    "        x=\"projection_y_coordinate\",\n",
+    "        ax=ax,\n",
+    "        marker=\"o\",\n",
+    "        alpha=0.2,\n",
+    "        label=\"cell {0}\".format(index),\n",
+    "    )\n",
+    "\n",
+    "ax.set_title(\"trajectories in the data\")\n",
+    "plt.legend()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f24bd05d",
+   "metadata": {},
+   "source": [
+    "The linking in `tobac` is performed with methods of the [trackpy](https://soft-matter.github.io/trackpy/v0.5.0/ )-library. Currently, there are two methods implemented, ['random'](https://soft-matter.github.io/trackpy/dev/generated/trackpy.link.html) and ['predict'](https://soft-matter.github.io/trackpy/v0.3.0/generated/trackpy.predict.NearestVelocityPredict.html). These can be selected with the `method` keyword. The default is 'random'."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "d16c1b0b",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Frame 69: 2 trajectories present.\n"
+     ]
+    }
+   ],
+   "source": [
+    "track = tobac.themes.tobac_v1.linking_trackpy(\n",
+    "    features, data, dt=dt, dxy=dxy, v_max=100, method_linking=\"predict\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1c5368b7",
+   "metadata": {},
+   "source": [
+    "## Defining a Search Radius\n",
+    "\n",
+    "If you paid attention to the input of the `linking_trackpy()` function you noticed that we used the parameter `v_max` there. The reason for that it would take a lot of computation time to check the whole data of a frame for the next position of a feature from the previous frame. Therefore the search is restricted to a circle around a position predicted by the [Crocker-Grier linking algorithm](http://dx.doi.org/10.1006/jcis.1996.0217), which is implemented in *trackpy*.\n",
+    "\n",
+    "The speed we defined before specifies such a radius timestep via\n",
+    "\n",
+    "$$\n",
+    "r_{max} = v_{max} \\Delta t.\n",
+    "$$\n",
+    "\n",
+    "By reducing the maximum speed fromm 100 m/s to 70 m/s we will find more cells, because a feature that is not assigned to an existing track will be used as a starting point for a new one:"
```

Comment:
```suggestion
    "By reducing the maximum speed from 100 m/s to 70 m/s we will find more cells, because a feature that is not assigned to an existing track will be used as a starting point for a new one:"
```